sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetIOSuite.scala

```diff
@@ -1461,10 +1462,7 @@ class ParquetIOSuite extends QueryTest with ParquetTest with SharedSparkSession
           vectorizedReader.initBatch(schema, partitionValues)
           vectorizedReader.nextKeyValue()
           val row = vectorizedReader.getCurrentValue.asInstanceOf[InternalRow]
-
-          // Use `GenericMutableRow` by explicitly copying rather than `ColumnarBatch`
-          // in order to use get(...) method which is not implemented in `ColumnarBatch`.
-          val actual = row.copy().get(1, dt)
```

Comment:
I can run this test with `null` without this change (L1467).

Reply:
That's correct. The `copy()` converts `row` from `ColumnarBatchRow` to `GenericInternalRow` and there null works. 
With this PR, `null` is directly supported in `ColumnarBatchRow.get()`

Reply:
Well, if this is a specifically required change, please do this change in a separate PR, @kazuyukitanimura .

Reply:
Thanks, actually we decided to close this PR.